tests/manage/mcg/test_bucket_policy.py

```diff
@@ -979,3 +979,63 @@ def test_multipart_with_policy(self, mcg_obj, bucket_factory):
         # Completing the Multipart Upload
         logger.info(f"Completing the MP Upload with on bucket: {bucket}")
         complete_multipart_upload(obc_obj, bucket, object_key, upload_id, uploaded_part)
+
+    @tier1
+    @pytest.mark.bugzilla("2210289")
+    def test_supported_bucket_policy_operations(self, mcg_obj, bucket_factory):
+        """
+        Test supported s3 bucket policies.
+        """
+        # Creating obc and obc object to get account details, keys etc
+        obc_name = bucket_factory(amount=1, interface="OC")[0].name
+        obc_obj = OBC(obc_name)
+
+        actions_list = [
+            "GetBucketObjectLockConfiguration",
+            "GetObjectRetention",
+            "GetObjectLegalHold",
+            "PutBucketObjectLockConfiguration",
+            "PutObjectRetention",
+            "GetObjectLegalHold",
```

Comment:
ListObjects 
ListMultipartUploads
GetObjectAttributes
ListObjectVersions
ListParts
PutObjectRetention
UploadPart
UploadPartCopy

are all the above operations covered here?